cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -246,6 +247,19 @@ public static boolean getBooleanParam(HttpServletRequest request, String paramet
     return parameterString == null ? defaultIfMissing : Boolean.parseBoolean(request.getParameter(parameterString));
   }
 
+  /**
+   * Get the long parameter parameter.
+   *
+   * @param request HTTP request received by Cruise Control.
+   * @param parameter Parameter to parse from the request.
+   * @param defaultIfMissing Default value to set if the request does not contain the parameter.
+   * @return The specified value for the parameter, or defaultIfMissing if the parameter is missing.
+   */
+  public static Long getLongParam(HttpServletRequest request, String parameter, @Nullable Long defaultIfMissing) {
```

Comment:
Should other `Long` parameters in this class also use of this function for parsing -- e.g. `EXECUTION_PROGRESS_CHECK_INTERVAL_MS_PARAM`?

Reply:
Yes, I applied this `getLongParam` method to `executionProgressCheckIntervalMs` and `replicationThrottle` and added unit tests